### PR TITLE
Convert erb templates to haml

### PR DESCRIPTION
**Pull request by eduardoj**
Replace ERB templates with HAML for main/, event_mailer/, ichain_notifier/ and layouts/ views.

Pairprogrammed by @bgeuken and @eduardoj.

### CODE REVIEW

File: src/api/app/views/layouts/webui/_flash.html.haml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#flash-messages.container_16
+  - unless flash.blank?
+    .ui-widget.grid_16.alpha.omega
+      - [:success, :error, :alert, :notice].each do |flash_type|
+      - if (flash[flash_type] && !flash[flash_type].empty?)
```
Comment (DavidKang) [inline src/api/app/views/layouts/webui/_flash.html.haml]: @eduardoj, it is bad indented

File: src/api/app/views/layouts/webui/_header.html.haml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/ Start: Header
+#header
+  #header-content.container_12
+    = link_to image_tag('obs-logo_small.png', height: 26), root_path, { id: 'header-logo' }
+    %ul#global-navigation
+      %li
```
Comment (DavidKang) [inline src/api/app/views/layouts/webui/_header.html.haml]: Why an empty %li? Maybe the content has to be into the %li :thinking:

File: src/api/app/views/layouts/webui/_sponsor.html.haml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+-# Don't show sponsor box
```
Comment (DavidKang) [inline src/api/app/views/layouts/webui/_sponsor.html.haml]: This file with a single comment is really needed?

File: src/api/app/views/layouts/webui/_flash.html.haml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+      - when :alert
+      - flash_icon = 'ui-icon-alert'
+      - flash_header = 'ui-state-error'
+      - when :success
+      - flash_icon = 'ui-icon-info'
+      - flash_header = 'ui-state-highlight'
+      - when :notice
+      - flash_icon = 'ui-icon-info'
+      - flash_header = 'ui-state-highlight'
+      - end
```
Comment (DavidKang) [inline src/api/app/views/layouts/webui/_flash.html.haml]: This `end` it not needed

File: src/api/app/views/layouts/webui/_header.html.haml (hunk 2 above)
Comment (bgeuken) [inline src/api/app/views/layouts/webui/_header.html.haml]: No idea, but it was there before (in the html)

Comment (DavidKang) [inline src/api/app/views/layouts/webui/_header.html.haml]: ok

**Comment by bgeuken** [comment]
We've updated the PR. The new version can be found here: #3395

**Pull request closed without merging**